Hangfire.Redis.StackExchange/RedisConnection.cs

```diff
@@ -150,13 +150,17 @@ public override bool GetSetContains([NotNull] string key, [NotNull] string value
             // Do not modify the original parameters.
             var storedParameters = new Dictionary<string, string>(parameters)
             {
-                { "Queue", invocationData.Queue },
                 { "Type", invocationData.Type },
                 { "Method", invocationData.Method },
                 { "ParameterTypes", invocationData.ParameterTypes },
                 { "Arguments", invocationData.Arguments },
                 { "CreatedAt", JobHelper.SerializeDateTime(createdAt) }
             };
+            
```

Comment:
I'm not sure why this change is not in the upstream. I saw that you merged that PR, but it still shows this as a change.